action/protocol/execution/evm/evm.go

```diff
@@ -250,7 +252,37 @@ func executeInEVM(evmParams *Params, stateDB *StateDBAdapter, gasLimit uint64) (
 	}
 	remainingGas += refund
 
-	return ret, evmParams.gas, remainingGas, contractRawAddress, evmErr != nil, nil
+	if isPostBering {
```

Comment:
This is so verbose. How about following?

```suggestion
errStatusCode = Unknown
switch evmErr {
	   case vm.ErrOutOfGas:
	        errStatusCode = FailureErrOutOfGas
	    ...
}
return ret, evmParams.gas, remainingGas, contractRawAddress, errStatusCode, nil 
```

Reply:
And you may consider write a errToErrStatusCode function for this